app/controllers/sessions_controller.rb

```diff
@@ -2,13 +2,16 @@ class SessionsController < ApplicationController
 
   def create
     user = User.find_by(username: params[:session][:username])
-    if user && user.authenticate(params[:session][:password])
+    if user && user.admin? && user.authenticate(params[:session][:password])
       session[:user_id] = user.id
-      session[:username] = user.username
-      flash[:success] = 'You have successfully logged in'
+      flash[:success] = 'You have successfully logged in.'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.